application/backend/src/main/resources/libs/bulma/components/image/dialog/.content.json

```diff
@@ -7,8 +7,8 @@
       "label": "General",
       "assetReference": {
         "sling:resourceType": "wcm/dialogs/components/assetreference",
-        "mimeTypes": ["image/*"],
-        "name": "assetReference",
+        "mimeTypes": ["image/*", "video/*"],
+        "name": "src",
```

Comment:
I'm not sure if you can do this. There is assetReference and the src in the model and I think that when we are checking if the selected asset is a video we should verify it by assetReference instead of src.